packages/api/app/Utils/technology_distribution.js

```diff
@@ -44,6 +44,11 @@ const distributeTechnologyToReviewer = async (technology) => {
 	const technologyCategoriesIds = technologyCategories.rows.map(
 		(technologyCategory) => technologyCategory.id,
 	);
+
+	const technologyRelatedUsers = await technology.users().fetch();
```

Comment:
Tenta fazer o select aqui pra não puxarmos tantos dados desnecessários.

```suggestion
	const technologyRelatedUsers = await technology.users().select('id').fetch();
```